bot.py

```diff
@@ -350,7 +1041,8 @@ async def embed(ctx):
         discord.embeds.Embed: A discord.embeds.Embed object with the parsed message.
     """
     input_string = """
-    {list_members test and 2 not 1}
+    {text_channel ogólny}
+    {voice_channel ogólny}
     """
     output_string = converting_string(ctx, input_string)
 
```

Comment:
```py
@bot.command()
async def server(ctx):
    """Reads the message from the chat and returns
embedded message with server stats.
```
Czy ta funkcja jest jeszcze używana?

Reply:
Na początku ją zostawiłem bo zawierała dużo funkcji, do których później wracałem, ale czy jest rzeczywiście potrzebna, pewnie nie. To do was pytanie, czy chcielibyście mieć funkcję, która pokazuje jakieś najważniejsze pre-zdefiniowane informacje. W teorii, wszystko powinni się dać zrobić przy użyciu funkcji /embed_creator.